cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis/CreateTestnetData.hs

```diff
@@ -253,6 +255,26 @@ runGenesisCreateTestNetDataCmd
 
     when (0 < numPools) $ writeREADME poolsDir poolsREADME
 
+    -- CC members
+    ccColdKeys <- forM [1 .. numCommitteeKeys] $ \index -> do
+      let committeeDir = committeesDir </> "cc" <> show index
+          vkeyHotFile = File @(VerificationKey ()) $ committeeDir </> "cc.hot.vkey"
+          skeyHotFile = File @(SigningKey ()) $ committeeDir </> "cc.hot.skey"
+          vkeyColdFile = File @(VerificationKey ()) $ committeeDir </> "cc.cold.vkey"
+          skeyColdFile = File @(SigningKey ()) $ committeeDir </> "cc.cold.skey"
+          hotArgs = CC.GovernanceCommitteeKeyGenHotCmdArgs ConwayEraOnwardsConway vkeyHotFile skeyHotFile
+          coldArgs = CC.GovernanceCommitteeKeyGenColdCmdArgs ConwayEraOnwardsConway vkeyColdFile skeyColdFile
```

Comment:
Can we not hardcode the era here?

Reply:
Yes we can if we make `create-testnet-data` only Conway onwards. Which makes sense, because it was introduced when we were developing conway. Right now we are incorrect, because nothing in `create-testnet-data` is parameterized on the era (everything is Shelley onwards, including for example DReps).

@Jimbo4350> what do you think about making `create-testnet-data` only Conway onwards?

Reply:
I would parameterize it on `Era era`.

Reply:
I created a follow-up issue to avoid mixing this change in this PR: https://github.com/IntersectMBO/cardano-cli/issues/962